contributor_licence_agreement.md

```diff
@@ -0,0 +1,195 @@
+CONTRIBUTOR LICENSE AGREEMENT
```

Comment:
Headings/ section formats are missing. 

See for how to apply different styles: https://guides.github.com/features/mastering-markdown/